app/javascript/components/widgets/twitter/tweet_content.jsx

```diff
@@ -23,8 +23,8 @@ export const Text = styled.div`
 `;
 
 const TwitterImage = styled.img`
-  height: ${props => (props.oneImg ? '487px' : 'auto')};
-  width: auto;
+  height: ${props => (props.oneImg ? '487px' : props.isMain ? '100%' : 'auto')};
```

Comment:
This nested ternary is causing a linter failure. Would it make sense to constrain the Wrapper to a specific height if there is only one image?

Reply:
Do you mean as like passing it as a style in the component?

Reply:
No, I meant if we constrain the ImageWrapper to a height based on `oneImg` you could have the TwitterImage only concern itself with `isMain` or not (assuming `isMain` is true when `oneImg` is true)

Reply:
Well imageWrapper is dependent on the amount of images, the size is either the full height or half the height